README.md

```diff
@@ -550,6 +550,26 @@ sent via this stream.
 See more details about these APIs in
 [grpc/\_client.py](src/python/library/tritonclient/grpc/_client.py).
 
+For gRPC AsyncIO requests, an AsyncIO task wrapping an `infer()` coroutine can
+be safely cancelled.
+
+```python
+  infer_task = asyncio.create_task(aio_client.infer())
+  await something_else
+  infer_task.cancel()
+```
+
+For gRPC AsyncIO streaming requests, `cancel()` can be called on the
+asynchronous iterator returned by `stream_infer()` API.
+
+```python
+  responses_iterator = aio_client.stream_infer()
+  responses_iterator.cancel()
+```
+
+See more details about these APIs in
+[grpc/\aio/\__init__.py](src/python/library/tritonclient/grpc/aio/__init__.py).
```

Comment:
this rendered weirdly, might not need backslashes, please double check it

Reply:
Good catch! Updated.